### PR TITLE
Add elected representative tabs to senators

### PR DESCRIPTION
**Pull request by kevin-verschaeve**
[ch11801]

### CODE REVIEW

File: templates/senator/_layout.html.twig [hunk 1]
```diff
@@ -73,6 +73,12 @@
                             </li>
                             {% endif %}
 
+                            {% if is_granted('ROLE_SENATOR') %}
```
Comment (ottaviano) [inline templates/senator/_layout.html.twig:76]: I see we check `is_granted('ROLE_SENATOR')` many times in this template. 
Could you refactor this for checking once this role, please?

Comment (kevin-verschaeve) [reply templates/senator/_layout.html.twig]: fixed :)

Comment (ottaviano) [reply templates/senator/_layout.html.twig]: the best will be: 
```twig
{% if is_senator %}
  <li>ITEM 1</li>
{% endif %}

{% if is_senator %}
  <li>ITEM 2</li>
{% endif %}
```
=> 
```twig
{% if is_senator %}
  <li>ITEM 1</li>
  <li>ITEM 2</li>
{% endif %}
```

Comment (kevin-verschaeve) [reply templates/senator/_layout.html.twig:76]: indeed, re fixed

**Pull request merged**